### PR TITLE
DSE Spark Master locator

### PR DESCRIPTION
**Pull request by noorul**

### CODE REVIEW

**Comment by noorul** [comment]
@velvia Can you take a look?

**Comment by velvia** [comment]
Sure I'll have a quick look later

-Evan
"Never doubt that a small group of thoughtful, committed citizens can change the world" - M. Mead

File: src/main/scala/spark.jobserver/masterLocators/dse/DseSparkMasterProvider.scala [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+ * limitations under the License.
+ */
+
+package spark.jobserver.masterLocators.dse
+
+import com.datastax.driver.core.{ResultSet, Session}
+import org.apache.spark.SparkConf
+import org.slf4j.LoggerFactory
+import scala.collection.JavaConverters._
+import com.datastax.spark.connector.cql.CassandraConnector
```
Comment (velvia) [inline src/main/scala/spark.jobserver/masterLocators/dse/DseSparkMasterProvider.scala]: Sort imports?

**Comment by velvia** [comment]
Looks fine to me overall.  I think this is the same code from the other PR right?

**Comment by noorul** [comment]
Yes this is the same pr

**Pull request merged**